packages/devtools_app/test/shared/landing_screen_test.dart

```diff
@@ -20,8 +18,38 @@ void main() {
       'Landing screen displays without error', const Size(2000.0, 2000.0),
       (WidgetTester tester) async {
     // Build our app and trigger a frame.
-    await tester.pumpWidget(wrap(LandingScreenBody()));
-    expect(find.text('Connect to a Running App'), findsOneWidget);
-    expect(find.text('App Size Tooling'), findsOneWidget);
+    await tester.pumpWidget(wrap(const LandingScreenBody()));
+    expect(find.byType(ConnectDialog), findsOneWidget);
+    expect(find.byType(ImportFileInstructions), findsOneWidget);
+    expect(find.byType(SampleDataDropDownButton), findsNothing);
+    expect(find.byType(AppSizeToolingInstructions), findsOneWidget);
+  });
+
+  testWidgetsWithWindowSize(
```

Comment:
Could we add a test to make sure this doesn't show in release mode?

Reply:
There isn't an easy way that I'm aware of to mock the value of kReleaseMode. The only way to test this would be to add another job to our github actions that runs the test suite in release mode, which may be a bit overkill for a test this small.